src/table/components/virtualized-table/types/index.ts

```diff
@@ -13,6 +13,15 @@ import { Totals } from '../hooks/use-totals';
 
 export type TotalsPosition = 'bottom' | 'noTotals' | 'top';
 
+interface Rect {
+  width: number;
+  height: number;
+}
+
+export interface TableRect extends Rect {}
+
+export interface StickyContainerRect extends Rect {}
```

Comment:
are these two necessary at this step? because they dont consider other types so cant't we use React instead?

Reply:
I had made a comment about this when creating the PR but appearently it ended up in a "pending" state.

Here it is:
"May seem a bit overkill to have two different interfaces with the same props. But I wanted to make a distinction that the actual values are different. Maybe there is a better way to do that?"

But looking at this again now. I think it fair to just use `rect`, as there are two different variables with proper names that describes the difference.